src/django/api/migrations/0011_add_statuschange_fields_to_facilityclaim.py

```diff
@@ -0,0 +1,155 @@
+# Generated by Django 2.0.13 on 2019-06-10 15:22
```

Comment:
Can this be combined with the 0010 migration without too much hassle. If it is not too much trouble I think it would be better to not have 2 migrations in the same PR that add a field and then immediately remove it.

Reply:
Yeah, these can be combined.